lib/Inst/Inst.cpp

```diff
@@ -447,11 +447,17 @@ const char *Inst::getKindName(Kind K) {
   case Hole:
     return "hole";
   case SAddO:
+    return "saddo";
```

Comment:
Are these operators supposed to be printed?

Reply:
no, i don't think so. The reason i added this here is to help in debugging where sometimes i want to print a DAG.